cpp/8puzzle.cpp

```diff
@@ -335,6 +337,19 @@ bool PuzzleState::IsSameState( PuzzleState &rhs )
 
 }
 
+// The 9 tiles positions can be encoded as digits

```

Comment:
Each tile is encoded as a digit (e.g., 125, 98, 1), put into the stream (e.g., 125981), then returned as the hash, correct? It looks like tiles are enums, so these numbers should all stay low. Is that correct?

Reply:
That is correct. 9 tile puzzle so it should be 0-8 only. I'll update the enum to be explicit.